src/components/Textarea/Textarea.style.ts

```diff
@@ -0,0 +1,61 @@
+import { styled } from 'styled-components';
+
+import { TextareaProps } from './Textarea.type';
+
+interface StyledTextareaProps {
+  $width?: TextareaProps['width'];
+  $height?: TextareaProps['height'];
+  $error?: TextareaProps['error'];
+}
+
+export const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+`;
+
+export const StyledTextarea = styled.textarea<StyledTextareaProps>`
+  width: ${({ $width }) => $width};
+  height: ${({ $height }) => $height};
```

Comment:
```suggestion
  width: ${({ $width }) => $width ?? "100%"};
  height: ${({ $height }) => $height ?? "auto"};
```

꼭 위방식이 아니더라도 기본값이 있으면 좋을듯하네요

Reply:
적용해두겠습니당~